<launch>
    <node name="pose_sensor" pkg="msf_updates" type="pose_sensor" clear_params="true" output="screen">

	<remap from="/MSF_Core/imu_state_input" to="/mav1/fcu/imu"/>
		<remap from="/MSF_Core/hl_state_input" to="/mav1/fcu/ekf_state_out"/>
		<remap from="/msf_core/pose_measurement" to="/mav1/vslam/pose"/>
		


	    <rosparam file="$(find ssf_updates)/pose_sensor_fix.yaml"/>
    </node>
</launch>
